<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Directorios</class>
 <widget class="QMainWindow" name="Directorios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Directorios</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbCrear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>210</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Crear Diretorio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRename">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>404</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Renombrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDelete">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>200</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Eliminar Directorios</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>364</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSalir">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRefresh">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refrescar la Lista</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCarpetas">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>325</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>200</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar archivos de texto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione la Carpeta para renombrarla</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Crea carpetas con numeracion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elimina los Directorios impares</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elimina los archivos de carpetas pares</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>541</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDelete_Direc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Directorio</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>650</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>711</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_3"/>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_4"/>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_5"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
